### PR TITLE
Update EarthEngine version to latest

### PR DESCRIPTION
**Pull request by janakdr**
EarthEngine now requests additional permissions, which Google.com corp accounts are not allowed to grant to third-party apps. As a workaround, use the token server if we get an error indicating that authentication failed due to a permissions error like this.

Images are loaded a bit differently now (`ee.MapLayerOverlay` is apparently deprecated), so had to modify layer_util.js. Is cleaner now.

Also noticed that after a minor version node upgrade (12.15-16), our tests started failing, so pinning at node v12.15. I think 12.16.1 should fix, but not important. More important is emailing test failures to GD when they happen.

### CODE REVIEW

File: docs/import/manage_disaster.html [hunk 1]
```diff
@@ -17,10 +17,11 @@
   <link href="../external_libs/jquery-ui-1.12.1.min.css" rel="stylesheet">
 
   <!-- Load debug Earth Engine JavaScript client library. -->
-  <!-- TODO(janakr): Switch back to compiled version in production. -->
+  <!-- TODO(janakr): Switch back to compiled version in production. Compiled
```
Comment (juliexxia) [inline docs/import/manage_disaster.html]: Still relevant TODO? When do you think this will be todone?

Comment (janakdr) [reply docs/import/manage_disaster.html]: See internal thread. They say this or next week?

**Comment by janakdr** [comment]
Updated to latest version of EE. Now actually works! Main change from previous version is in layer_util.js.

File: docs/layer_util.js [hunk 2]
```diff
@@ -268,14 +267,26 @@ function showOverlayLayer(overlay, index, map) {
 function resolveOnEeTilesFinished(layerDisplayData, resolve) {
   if (layerDisplayData.tileCallbackId) {
     layerDisplayData.overlay.removeTileCallback(
-        layerDisplayData.tileCallbackId);
+        layerDisplayData.tileCallbackId[0]);
```
Comment (juliexxia) [inline docs/layer_util.js]: This method seems to be more than a wrapper now - update doc to reflect?

File: cypress/integration/unit_tests/checkbox_test.js [hunk 3]
```diff
@@ -305,8 +305,10 @@ describe('Unit test for toggleLayerOn', () => {
         })
         .then(() => {
           expect(loadingFinishedStub).to.be.calledOnce;
+          expect(map.overlayMapTypes).to.have.length(4);
           overlay = map.overlayMapTypes.getAt(3);
           expect(overlay).is.not.null;
+          expect(overlay).to.not.be.undefined;
```
[the remark this answers is not in the document]
Comment (janakdr) [reply cypress/integration/unit_tests/checkbox_test.js]: Done.

File: docs/layer_util.js (hunk 2 above)
Comment (janakdr) [reply docs/layer_util.js]: Done, and fixed stale reference in doc. Thanks!

File: docs/import/ee_utils.js [hunk 4]
```diff
@@ -19,6 +19,7 @@ async function listEeAssets(path) {
     let assets;
     const listResult =
         await ee.data.listAssets(path, {page_token: nextPageToken}, () => {});
+    console.log(listResult.assets);
```
Comment (juliexxia) [inline docs/import/ee_utils.js]: rm

Comment (janakdr) [reply docs/import/ee_utils.js]: Oops, done!

**Pull request merged**